# errno.ETIMEDOUT

Connection timed out.

This error is mapped to the exception [`TimeoutError`](/exceptions/TimeoutError.md).

### Syntax

```python
errno.ETIMEDOUT
```
